monitor.h

```diff
@@ -1,3 +1,11 @@
 #pragma once
 
-int vitalsOk(float temperature, float pulseRate, float spo2);
+// Define constants for the upper and lower limits
+const double TEMPERATURE_LOWER_LIMIT = 95.0;

```

Comment:
Nice names, but see if the unit is needed. E.g., Fahrenheit for temperature, due to possible ambiguity. Unit not needed for SPO2